### ISSUE TITLE
Context Menu is empty

### ISSUE THREAD

**Issue by knotvoid** [benbasson/contextsearch#11]
The addon doesn't work well. 
The item in context menu on Firefox Nightly 41.0a1 is empty. 
If you need a Screenshot to see it I will take one.

**Comment by benbasson** [comment]
I'm using Context Search - fully working - on Firefox 41 release build. I would imagine this was a nightly build issue? Let me know if you're still having problems on a release build and we can reopen this issue. I don't usually support nightlies.

**Comment by Vangelis66** [comment]
Hi Ben - many thanks for your addon and constant efforts to maintain it!

> I would imagine this was a nightly build issue?

Using latest Nightly 44.0a1, the situation is pretty much 
as described by the OP:

![e10s-on](https://cloud.githubusercontent.com/assets/9669492/10455971/4f0dde72-71c7-11e5-9386-6eb39f60e730.jpg)

However, this is because your addon hasn't been made 
e10s compatible yet :-( and from Nightly 40.0a1, e10s 
have been turned on by default: 

Multiprocess Windows 	1/1 (default: true), i.e. 
browser.tabs.remote.autostart.2;true

If one turns off e10s manually, either via the GUI option:
about:preferences -> General -> uncheck "Enable multi-process Nightly"
or by preffing off browser.tabs.remote.autostart.2 in about:config, 
followed by a Fx restart, then Context Search 0.6.3.1 can be used normally: 

![e10s-off](https://cloud.githubusercontent.com/assets/9669492/10456591/d059a0c6-71ca-11e5-9907-3ba614b0a3b5.jpg)

> I don't usually support nightlies.

I can respect this totally, however many (but not all) features 
debuting in the Nightly branch make it eventually after some months 
into the Release branch; my humble opinion is that you being an 
addon developer, you should keep a watchful eye on what is happening 
on the pre-release branches; you can read (if not already) more 
about e10s at:
https://wiki.mozilla.org/Electrolysis

Let's all hope that by the time Electrolysis makes it 
to the release branch, there will still be addons 
as we know them already:

http://liliputing.com/2015/08/firefox-to-gain-support-for-chrome-add-ons-and-lose-support-for-legacy-xul-plugins.html
https://blog.mozilla.org/addons/2015/08/21/the-future-of-developing-firefox-add-ons/

Many regards from Greece...

**Comment by benbasson** [comment]
Hi - thanks for your comments. I do test with nightly builds in preparation for releases, but usually not over two months before the projected release build. 

To provide some background, I've been doing this for over a decade now, and I used to actively use and test nightly builds. In the past, I used to switch to release branch builds early on and test my add-ons, since hardly any serious breaking changes would be introduced on release branches. Since Mozilla brought in the "rapid release" schedule, nobody seems to care about that any more and I don't have the patience or time to keep up with it all.

In short: I would love to do more to support nightly builds, but unless Mozilla reverse their rapid release schedule decision and start giving add-on developers more stability, I'm not going to.
